datasets/ade_corpus_v2/ade_corpus_v2.py

```diff
@@ -0,0 +1,272 @@
+}
+"""
+
+_DESCRIPTION = """\
+ ADE-Corpus-V2  Dataset: Adverse Drug Reaction Data.
+ This is a dataset for Classification if a sentence is ADE-related (True) or not (False) and Relation Extraction between Adverse Drug Event and Drug.
+ DRUG-AE.rel provides relations between drugs and adverse effects.
+ DRUG-DOSE.rel provides relations between drugs and dosages.
+ ADE-NEG.txt provides all sentences in the ADE corpus that DO NOT contain any drug-related adverse effects.
+"""
+
+_DOWNLOAD_URL = "https://raw.githubusercontent.com/trunghlt/AdverseDrugReaction/master/ADE-Corpus-V2/{}-{}.{}"
+
+# Different usage configs/
+configs = {
+    "classifiaction": "Ade_corpos_v2_classificaion",
+    "RE_ade": "Ade_corpos_v2_drug_ade_relation",
+    "RE_dosage": "Ade_corpos_v2_drug_dosage_relation",
+}
+
+
+class ADE_Corpus_V2Config(datasets.BuilderConfig):
+    """BuilderConfig for ADE_Corpus_V2."""
+
+    def __init__(self, **kwargs):
+        """BuilderConfig for ADE_Corpus_V2.
+        Args:
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(ADE_Corpus_V2Config, self).__init__(**kwargs)
+
+
+class ADE_Corpus_V2(datasets.GeneratorBasedBuilder):
+    """ADE_Corpus_V2 Dataset: Adverse Drug Reaction Data for Classification and Relation Extraction tasks ."""
+
+    BUILDER_CONFIGS = [
+        ADE_Corpus_V2Config(
+            name="Ade_corpos_v2_classificaion",
+            version=datasets.Version("1.0.0"),
+            description="ADE_Corpus_V2 Dataset for Classification if a sentence is ADE-related or not.",
+        ),
+        ADE_Corpus_V2Config(
+            name="Ade_corpos_v2_drug_ade_relation",
+            version=datasets.Version("1.0.0"),
+            description="ADE_Corpus_V2 Dataset for Relation Extraction between Adverse Drug Event and Drug.",
+        ),
+        ADE_Corpus_V2Config(
+            name="Ade_corpos_v2_drug_dosage_relation",
+            version=datasets.Version("1.0.0"),
+            description="ADE_Corpus_V2 Dataset for Relation Extraction between Drug dosage and Drug.",
+        ),
+    ]
+
+    def _info(self):
+
+        if self.config.name == configs["classifiaction"]:
+            features = datasets.Features(
+                {
+                    "text": datasets.Value("string"),
+                    "label": datasets.features.ClassLabel(names=["Related", "Not-Related"]),
```

Comment:
```suggestion
                    "label": datasets.features.ClassLabel(names=["Not-Related", "Related"]),
```
usually we have the positive label at index 1